runtime/src/lib.rs

```diff
@@ -548,7 +548,13 @@ parameter_types! {
 	pub const TipCountdown: BlockNumber = 1 * DAYS;
 	pub const TipFindersFee: Percent = Percent::from_percent(20);
 	pub const TipReportDepositBase: Balance = 1 * DOLLARS;
-	pub const TipReportDepositPerByte: Balance = 10 * MILLICENTS;
+	pub const DataDepositPerByte: Balance = 1 * CENTS;
+	pub const BountyDepositBase: Balance = 1 * DOLLARS;
+	pub const BountyDepositPayoutDelay: BlockNumber = 8 * DAYS;
+	pub const BountyUpdatePeriod: BlockNumber = 90 * DAYS;
```

Comment:
I would recommend this number be lower. This configuration basically sets how often a curator must report back an update to the network. I suggested it should be 30 days on Polkadot, but Raul wanted to be a bit more conservative.

I think 3 months per update from a curator is just too long.

Reply:
It's just a call to `extend_bounty_expiry` that the curator can call at any time, right? So how about we set it to lower? Something like 16 days?

Reply:
Yeah 2-4 weeks all makes sense to me.